HW20/hosts.yaml

```diff
@@ -0,0 +1,10 @@
+all:
+  children:
```

Comment:
to make a bit "smaller" try to define groups after you specify groups with hosts.
For example
```
all:
  children:
    testhost1:
      hosts:
        172.30.128.3
    testhost2:
      hosts:
        172.30.128.4
    centos:
      hosts:
        172.30.128.5
    mix:
      children:
        testhost1:
        centos:

Reply:
In my case it will give more lines.